src/app/(BottomTabNavigation)/AllCases/CaseScreen/[caseUid].tsx

```diff
@@ -25,6 +27,21 @@ function CaseScreen() {
     setIsLoading(false);
   };
 
+  const onShare = async () => {
+    const addCaseUrl = Linking.createURL('?event=addCase&', {
+      queryParams: { caseUid },
+    });
+    console.log(addCaseUrl);
+    try {
+      await Share.share({
+        //message: 'ADD CASE',
+        url: `${addCaseUrl}/`,
```

Comment:
Out of curiosity, is there anyway to customize how the link looks when shared with someone? Sometimes apps will attach an image or additional text to the link so users don't have to see the entire link.